client/src/pages/ManageBookings/ManageBooking.tsx

```diff
@@ -0,0 +1,133 @@
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import Typography from '@mui/material/Typography';
+import useStyles from './useStyles';
+import { Grid } from '@mui/material';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import Avatar from '@mui/material/Avatar';
+import userImage from '../../images/userProfilePic.jpeg';
+import IconButton from '@mui/material/IconButton';
+import SettingsIcon from '@mui/icons-material/Settings';
+import List from '@mui/material/List';
+import Calendar from 'react-calendar';
+import 'react-calendar/dist/Calendar.css';
+import './bookingCalendar.css';
+
+function Booking(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Grid display={'flex'} flexDirection={'column'} className={classes.bookingComponent} flexWrap={'nowrap'}>
+      <Grid container direction="row" justifyContent="space-between" className={classes.bookings} alignItems={'center'}>
+        <Typography variant="h6" className={classes.bookingTitleDate}>
+          5 April 2020, 10-12 AM
+        </Typography>
+        <IconButton aria-label="settings" className={classes.bookingIcon}>
+          <SettingsIcon />
+        </IconButton>
+      </Grid>
+      <Grid display={'flex'} flexDirection={'row'} width={'90%'} mx="auto" className={classes.bookingContainer}>
+        <Grid container direction={'row'} justifyContent={'flex-start'}>
+          <Avatar alt="userImage" src={userImage} variant={'circular'} className={classes.bookingAvatar} />
+          <Typography variant="h6" className={classes.bookingAvatarName}>
+            Norma Byers
+          </Typography>
+        </Grid>
+        <Typography variant="h6" className={classes.bookingAvatarStatus}>
+          ACCEPTED
```

Comment:
Shouldn't explicitly use uppercase as screen readers will read this as an abbreviation. Would be a poor UX for users with screen readers.